README.md

```diff
@@ -60,30 +60,45 @@ Empty tags will have a value of nil.
 
 -----
 
-There is a dependency on Erlsom to parse xml then converts the 'simple_form' structure into a map.
+There is a dependency on Erlsom to parse XML then converts the 'simple_form' structure into a map.
 
-I prefer Erlsom because it is the best documented erlang xml parser and because it mentions that it does not produce new atoms during the scanning.
+I prefer Erlsom because it is the best documented erlang XML parser and because it mentions that it does not produce new atoms during the scanning.
 
 See tests for some example usage.
 
 ----
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
+The package can be installed as:
 
-  1. Add `elixir_xml_to_map` to your list of dependencies in `mix.exs`:
+1.  Add `:elixir_xml_to_map` to your list of dependencies in `mix.exs`:
 
-```elixir
-def deps do
-  [{:elixir_xml_to_map, "~> 2.0"}]
-end
-```
+    ```elixir
+    def deps do
+      [{:elixir_xml_to_map, "~> 2.0"}]
+    end
+    ```
 
-  2. Ensure `elixir_xml_to_map` is started before your application:
+2.  Ensure `:elixir_xml_to_map` is started before your application:
 
-```elixir
-def application do
-  [applications: [:elixir_xml_to_map]]
-end
-```
+    ```elixir
+    def application do
+      [applications: [:elixir_xml_to_map]]
+    end
+    ```
+
+
+## License
+
+Copyright (C) 2016-present, Homan Chu
```

Comment:
Name spelled, Homan Chou

Reply:
> Name spelled, Homan Chou

@homanchou Fixed! Sorry about that. :smile: